pages/apim/3.x/event-native-apim/event-native-apim-introduction.adoc

```diff
@@ -0,0 +1,82 @@
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+== Event-driven asynchronous APIs
+
+Most developers are familiar with synchronous APIs, where the client sends a request to a server, and the server sends a response back. Every time the client needs new data, it has to send a request. You can think of synchronous APIs as a 'client pull' model. RESTful APIs are a type of synchronous API.
+
+In contrast, asynchronous APIs are event-driven and allow you to stream data. Asynchronous APIs can use callbacks, such as webhooks, if the requested data is not available at the time of the request. You can think of asynchronous APIs as a 'server push' model. AsyncAPIs are a type of asynchronous API.
+
+== Event-native API management
+
+With the rise of cloud computing, many vendors started to build their solutions for the cloud. Rather than taking on-premises software and converting it to work as SaaS, it was built from the ground to to be cloud-based. These solutions are said to be cloud-native.
+
+Event-native API management is similar. Rather than taking a synchronous API management solution and converting it to handle event-driven APIs, an event-native API management solution is built from the ground up to handle event-driven APIs.
+
+== Gravitee's event-native API management
+
+Gravitee's API management solution can run in event-native mode, and it can support all asynchronous, streaming, and event-driven API use cases. Of course, Gravitee's API management solution also supports synchronous use cases.
+
+Gravitee can act as a bridge between synchronous and asynchronous environments.
+
+For example, Gravitee can subscribe to a Kafka topic and then allow filtered HTTP GET subscriptions from clients as shown in the following diagram.
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-gateway.png %}[]
+
+== BETA Features
+
+The following features are available in the BETA release.
+
+=== HTTP POST and HTTP GET entrypoints
+
+The HTTP POST and HTTP GET entrypoints are Community Edition / OSS plugins that enable external clients to publish and receive data via standard HTTP POST / HTTP GET requests, and ease the ingestion of data into event-orientated backends like Kafka.
+
+With the HTTP POST workflow, a client application running on HTTP can post content to the Gravitee Gateway. The Gateway can then take that content and push it over to Kafka as a message that Kafka can ingest. The Gravitee Gateway acts as a protocol mediation layer in this context. This enables API owners to quickly set up a sync-to-async integration.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-http-post.png %}[]
+
+The HTTP POST entrypoint can be used in combination with specific Async API endpoints in different ways, depending on what publish and subscribe mode actions are possible between the two when using a connector. 
```

Comment:
```suggestion
The HTTP POST entrypoint enables you to push messages to any supported event broker via a "sync call" (post) - it can be used in combination with specific Asynchronous API endpoints in different ways, depending on what publish and subscribe mode actions are possible between the two when using a connector. 
```